node/src/types/available_block_range.rs

```diff
@@ -45,6 +45,16 @@ impl AvailableBlockRange {
     pub fn contains(&self, height: u64) -> bool {
```

Comment:
(That comment in line 11 should probably say something about "error" and "low > high"…)

Reply:
Thanks - fixed in [9fcb768](https://github.com/casper-network/casper-node/pull/2844/commits/9fcb76807c46450853f6ca6d04e054031b4426e3).